docs/user/ml/index.asciidoc

```diff
@@ -18,7 +18,17 @@ image::user/ml/images/ml-data-visualizer-sample.jpg[Data Visualizer for sample f
 
 experimental[] You can also upload a CSV, NDJSON, or log file (up to 100 MB in 
 size). The *Data Visualizer* identifies the file format and field mappings. You 
-can then optionally import that data into an {es} index.  
+can then optionally import that data into an {es} index.
+
+You need the following permissions to use the Data Visualizer with file upload:
+
+* cluster privileges: `monitor`, `manage_index_pipelines`, 
+`manage_index_templates`
+* index privileges: `manage`, `index`
```

Comment:
You should add `read` to the list here. Without it, the user wouldn't be able to read the data they ingested.